### ISSUE TITLE
Add Documentation, Support and Dashboard links to Service instance details pages

### ISSUE THREAD

**Issue by ncameronbritt** [openshift/origin-web-console#2984]
We show these links on the wizard results and in the overview, but not on the details pages.

Does it make sense (at least as a quick fix) to put the `View Documentation`, `Get Support`, etc links underneath the Description, just as we do for the expanded rows in the Overview?

![screen shot 2018-05-02 at 3 44 01 pm](https://user-images.githubusercontent.com/8974593/39545599-b206c4b2-4e1f-11e8-8f6a-093de1615ba7.png)

cc @openshift/team-ux-review

**Comment by serenamarie125** [comment]
@ncameronbritt makes sense for them to be in the wizards, overview and detail page.

I'm running on Online, and don't see the same view you are - were these links added to the overview this release?

**Comment by ncameronbritt** [comment]
@serenamarie125 No, they've been there. Online is not using the service catalog, that's why you're not seeing them.

**Comment by beanh66** [comment]
@ncameronbritt makes sense to me!
